packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -105,12 +111,36 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
     isMuting,
   ]);
 
+  const shouldDisplayTrimButton = useMemo(
+    () => isSingleElement && isVideoTrimEnabled,
+    [isSingleElement, isVideoTrimEnabled]
+  );
+
   const buttonText = useMemo(() => {
     return isMuting
       ? __('Removing audio', 'web-stories')
       : __('Remove audio', 'web-stories');
   }, [isMuting]);
 
+  const { isEditing, setEditingElementWithState, clearEditing } = useCanvas(
+    ({
+      state: { isEditing },
+      actions: { setEditingElementWithState, clearEditing },
+    }) => ({
+      isEditing,
+      setEditingElementWithState,
+      clearEditing,
+    })
+  );
+
+  const handleTrim = useCallback(() => {
+    if (isEditing) {
+      clearEditing();
+    } else {
```

Comment:
This does have the weird side-effect, that if you double-click a video to edit the frame/scale and then press the "trim" button, the trim button just exits edit mode without entering trim mode. But that's a minor concern we can always discuss later.

We can also discuss whether this button should appear in a pressed state when the trim mode is active, so it's clear that it's a toggle button?

Reply:
> We can also discuss whether this button should appear in a pressed state when the trim mode is active, so it's clear that it's a toggle button?

If it's a toggle then yes, it should have active mode :)

Reply:
Hm, we don't actually have a toggle button design for a non-icon button - [all of the existing variants](https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-components-button--toggle-buttons) look misplaced in context if I insert them. So I will leave this as-is for now, and we can discuss it when fine-tuning the UX at the end.